.github/workflows/website-deploy.yaml

```
@@ -0,0 +1,60 @@
+name: websites:deploy
+
+on:
+  workflow_call:
+    secrets:
+      WEBPLANNER_WEBHOOK_URL:
+        description: Webplanner webhook URL
+        required: true
+      SFTP_KEYFILE:
+        description: SFTP SSH key content
+        required: true
+      RCLONE_CONF:
+        description: Env specific config for file sync
+        required: true
+      WEBHOOK_TOKEN:
+        description: Token required for webplanner curl
+        required: true
+
+jobs:
+  build:
+    env:
+      commitmsg: ${{ github.event.head_commit.message }}
+      user_email: ${{ github.event.pusher.email }}
+      repo_name: ${{ github.event.repository.name }}
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/clickdealer-themes/rclone-docker:latest
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: checkout web scripts
+        uses: actions/checkout@v3
+        with:
+          repository: clickdealer-themes/websites #pull down contents of repo containing scripts for web
+          token: ${{ secrets.GITHUB_TOKEN }}
```

this token won't have permission to pull `clickdealer-themes/websites` so we'll need to probably request a token to be passed in as a secret and use that instead (this will need adding to the section at the top).

```suggestion
          token: ${{ secrets.WEBSITES_DEPLOY_PAT }}
```